gaia-fast-list.js

```diff
@@ -173,22 +177,18 @@ var GaiaFastListProto = {
         color: var(--title-color);
       }
 
+      ::content .gfl-section {
+        position: relative;
+        padding-top: 20px;
```

Comment:
do we want this even on the first section?
(note: we could actually use `:first-child` on sections since they're ordered in the dom)